### ISSUE TITLE
Handling multiple schedulers

### ISSUE THREAD

**Issue by colsond** [hep-gc/cloudscheduler#6]
Currently the job ads are stored in the database using the primary key `GlobalJobId` This may introduce a problem for the poller threads.

Firstly I am unsure if condor will guarantee uniqueness across multiple scheduler threads. If it doesn't we will need to introduce a second part of the key that identifies which scheduler thread the job is from.

Secondly, even if condor guarantees uniqueness across multiple schedulers both the cleanup algorithm and condor command executer will not know which jobs are from "it's" scheduler and may try to remove or execute condor hold on jobs that are from another scheduler.

In conclusion we need to devise a method for naming the scheduler threads such that their data can be accessed individually in the database.

My first two thoughts were:
1. Specify manually through the poller configuration files. This may be difficult to manage with many clouds being managed by cloud scheduler and could lead to malfunctions if there is misconfiguration.
2. Attempt to find some unique identifier already in the condor data. This would be the best case as there could be no human error but it is not clear at this point if such an identifier already exists.

**Comment by mhpx** [comment]
The hostname is included so it shouldn't be a problem i.e.
GlobalJobId = "bellecs.heprc.uvic.ca#465302.9#1510686938"

**Comment by colsond** [comment]
Ah yes of course, I was just looking at that too. I will see if i can update the query to only return results from the local host. Hopefully it will be painless to retrieve the host name on the local machine for comparison, if not we can always add it to the configuration.

**Comment by colsond** [comment]
Addressed in commit 0e31e041dc10290234e8b060a66a221fb40f936f
Changes are untested.
